### ISSUE TITLE
Missing a font, can't run locally

### ISSUE THREAD

**Issue by ccurtis7** [freakboy3742/traveltips#9]
I've been looking into using BeeWare in some of my Python courses I teach, really exciting stuff here. I ran through the tutorial materials, which worked fine. I was looking for some more complicated apps to take it a step further (it looks like you have that in the works), when I found this on social media. 

I tried creating a build locally. When I run `briefcase dev`, I get the following error:

    Traceback (most recent call last):
    File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\fonts.py", line 16, in __init__
    font = _FONT_CACHE[self.interface]
    KeyError: <Font: bold 16pt Helvetica>

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
    File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\app.py", line 144, in run_appself.create()
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\app.py", line 65, in create
        self.interface.startup()
      File "C:\Users\koolk\Dropbox (NSC)\NSC\spring2022\cs220\W13Apps\traveltips\src\traveltips\app.py", line 125, in startup
        text_align=RIGHT,
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga\widgets\label.py", line 20, in __init__
        self._impl = self.factory.Label(interface=self)
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\widgets\base.py", line 12, in __init__
        self.interface.style.reapply()
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\travertino\declaration.py", line 88, in reapply
        self.apply(style, getattr(self, style))
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga\style\pack.py", line 89, in apply
        weight=self.font_weight
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga\style\applicator.py", line 24, in set_font
        self.widget._impl.set_font(font)
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\widgets\label.py", line 20, in set_font
        self.native.Font = value._impl.native
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga\fonts.py", line 25, in _impl
        self.bind(None)
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga\fonts.py", line 30, in bind
        self.__impl = factory.Font(self)
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\fonts.py", line 22, in __init__
        font_family
      File "C:\Users\koolk\Anaconda3\envs\beeware\lib\site-packages\toga_winforms\libs\fonts.py", line 59, in win_font_style
        if weight.lower() == "bold" and font_family.IsStyleAvailable(
    AttributeError: 'NoneType' object has no attribute 'IsStyleAvailable'

It looks like there is a missing font somewhere in toga? I tried updating toga to see if that would fix it, but nothing seemed to change. Any help would be appreciated.

**Comment by freakboy3742** [comment]
Thanks for the report. This is probably related to the new Toga release that we published last night, which included a couple of new features related to fonts. I'll take a look and see if I can work out what is going on.

**Comment by freakboy3742** [comment]
It turns out the problem is actually the other way around; the version of Toga that is pinned in `pyproject.toml` is 0.3.0dev20; we released 0.3.0dev30 last night, which corrects this font handling problem. 

However, the new release has a different problem related to the handling of min/max values. I've corrected the problem in beeware/toga#1401. If you want to try using this fix:

* Clone the beeware/toga repository somewhere locally
* Modify your `pyproject.toml`:
  - uncomment line 18 (`../../toga/src/core`), making it a reference to your toga checkout    
  - uncomment line 30 (`../../toga/src/winforms`), making it a reference to your toga checkout
  - comment out line 31 (1toga-winforms==0.3.0.dev20`) 

The final `pyproject.toml` should look like (in part):
```
...
requires = [
    '../../toga/src/core'
]
...
[tool.briefcase.app.traveltips.windows]
requires = [
    '../../toga/src/winforms'
    # 'toga-winforms==0.3.0.dev30'
]
```

Then run `briefcase dev -d` and `briefcase update -d` to make sure the new dependencies are picked up.

**Comment by freakboy3742** [comment]
This has been resolved by more recent Toga updates; the project configuration as of b59c01c works in my testing.
